book/src/main/scalatex/handson/PublishingModules.scalatex

```diff
@@ -127,7 +127,7 @@
       Cross-cross build it! You can use @hl.scala{crossScalaVersions} in your @hl.scala{crossProject} to build a library that works across all of {Scala.js, Scala-JVM} X {2.10, 2.11}. Many existing libraries, such as @lnk("Scalatags", "https://github.com/lihaoyi/scalatags") or @lnk("uTest", "https://github.com/lihaoyi/utest") are published like that.
 
   @p
-    Now that you've gotten your code cross-compiling to Scala.js/Scala-JVM, the sky's the limit in what you can do. In general, although a large amount of your Scala-JVM code @i{does} deal with files or networks or other Scala-JVM-speciic functionality, in most applications there is a large library of helpers which don't. These could easily be packaged up into a cross-platform library and shared with your front-end Scala.js (or even pure-Javascript!) code.
+    Now that you've gotten your code cross-compiling to Scala.js/Scala-JVM, the sky's the limit in what you can do. In general, although a large amount of your Scala-JVM code @i{does} deal with files or networks or other Scala-JVM-specific functionality, in most applications there is a large library of helpers which don't. These could easily be packaged up into a cross-platform library and shared with your frontend Scala.js (or even pure JavaScript!) code.
```

Comment:
Elsewhere you opted for to remove the dash in "Scala-JVM", here and in surrounding paragraphs you have maintained it.  Just a consistency issue.  I am not sure if the Scala community has adopted a standard yet; it seems like they should to help unify the platform since there is currently Scala/JVM, Scala/Android, Scala.js, and Scala Native which are not all equally known.

Also, you use two words for "back end" so it would probably be best to use two words for "front end" as well.